pypeit/coadd2d.py

```diff
@@ -996,7 +1087,7 @@ def reference_trace_stack(self, slitid, offsets=None, objid=None):
             #. ``offsets``: We stack about the center of the slit for
                the slit in question with the input offsets added
 
-            #. ``ojbid``: We stack about the trace ofa reference
+            #. ``ojbid``: We stack about the trace of a reference
```

Comment:
This variable name is misspelled, should be oBjid, not oJbid

Reply:
Agreed, can you fix?